docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -350,11 +363,86 @@ This request lists services in the apimlId domain.
         "services": [
             {
                 "status": "UP",
-                "customMetadata": {},
+                    "customMetadata": {
+                 "zos.sysname": "sys1",
+		 "zos.sysplex": "sysplex"
+				        },
                 "apiId": [
         "zowe.apiml.gateway"
 ],
                 "serviceId": "gateway"
+            },
+            {
+             "status": "UP",
+                "customMetadata": {
+                 "zos.sysname": "sys1",
```

Comment:
Maybe different sysnames for different APIMLs? To create better visibility that they are on different systems.